### PR TITLE
new bj_ma.md file

### PR DESCRIPTION
**Pull request by biancajiang**
added new folder bj_ma
added new file bj_ma/bj_ma.md

### CODE REVIEW

**Comment by biancajiang** [comment]
@flam312  pls review

**Comment by biancajiang** [comment]
@eclancy68  howdy!

**Comment by biancajiang** [comment]
@tanndev  hey!

File: bj_ma/bj_test.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Massachussets
+
+**Important Note:** We will do our best to get through everything in this outline, but depending on the class, we may not spend as much time on some of the more advanced topics at the end. We will touch on them all, though! :smile: 
+
+### Places to eat
```
Comment (jamie-tm) [inline bj_ma/bj_test.md:5]: Outtakes has the best coffee in the building. ☕

File: bj_ma/bj_test.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+- Introducing GitHub
+  - Key GitHub Features
+  - The GitHub Ecosystem
```
Comment (biancajiang) [inline bj_ma/bj_test.md:9]: what's this?

File: bj_ma/bj_test.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+- Exploring a Repository
+- Using Issues
+
+### Places to stay
+
+- Overview of the GitHub Flow
```
Comment (karljanusz) [inline bj_ma/bj_test.md:15]: I love that place! Easily top 10 in Massachusetts!

**Pull request merged**